cmd/utils/history_test.go

```diff
@@ -170,7 +170,7 @@ func TestHistoryImportAndExport(t *testing.T) {
 		db2.Close()
 	})
 
-	genesis.MustCommit(db2, triedb.NewDatabase(db, triedb.HashDefaults))
+	genesis.MustCommit(db2, triedb.NewDatabase(db2, triedb.HashDefaults))
```

Comment:
Sure about this? Looks like it was intentional. Commit from `db` into `db2`?

Reply:
If it was not intentional, it would be better to just re-assign `db`, on line `165`.

Reply:
It was an issue in the test. We should use the same db instance for both key-value store and triedb.